hetu-docs/en/admin/properties.md

```diff
@@ -922,3 +922,64 @@ helps with cache affinity scheduling.
 > After the configured time elapsed and no response received, then client connection consider that to be failure in submission of request.
 >
 > (Note: this parameter should be configured with higher time when in high load environment)
+
+## Gossip Protocol Configurations for Failure Detection
+
+### `failure-detection-protocol`
+
+>- **Type:** String
+>- **Default value:** `heartbeat`
+> 
+> This property defines the type of failure detector in use. Default configuration is `heartbeat` failure detector.
+> Gossip protocol can be enabled by specifying this parameter in `config.properties` file, with the value `gossip`.
+> All nodes (i.e. coordinator as well as workers) in a cluster should have this property specified in their respective `etc/config.properties` file.
+
+### `failure-detector.heartbeat-interval`
```

Comment:
naming needs standardization

Reply:
this is existing config from heartbeat failure detector.